docs/DevGuide/Brigand.md

```diff
@@ -1819,13 +1819,11 @@ Avoid the `*_wrapper` functions in favor of using the class directly.
 
 \par
 Brigand provides functions to produce the `boost::fusion` types `deque`,
-`list`, `set`, and `vector`, as well as `boost::variant`.
-\snippet Test_TMPLDocumentation.cpp boost_integration
-
-\note
-These functions are unavailable if `BRIGAND_NO_BOOST_SUPPORT` is defined, as is
-the case in SpECTRE.
-
+`list`, `set`, and `vector`, as well as `boost::variant`. Because we use brigand
+instead of boost::fusion and because `std::variant` replaces `boost::variant`,
+these are not available in SpECTRE. Converting a sequence to a `std::variant` to
+a is done as
+\snippet Test_TMPLDocumentation.cpp std_variant_integration
```

Comment:
Drop this last sentence and the accompanying example.  The `make_std_variant_over` metafunction isn't used anywhere, and it does a bunch of stuff that normal Brigand functions don't (removes duplicates, bans `void*` for some reason).

If you must keep it, fix the grammar.

Reply:
I just dropped it. I agree, it's sufficiently different and we don't use it right now. I think it used to be used for the DataBox variant when we still had that.